pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,117 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Executing = "Executing"
+	Running   = "Running"
+	Pending   = "Pending"
+	Canceled  = "Canceled"
+	Blocked   = "Blocked"
+	Ready     = "Ready"
+	Deleted   = "Deleted"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ succeeded, failed, Executing, Canceled]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
```

Comment:
will it be shown as openshift or with some capital letter?
same comment about all provider comments.

Reply:
it will be displayed in the way we keep the providers name in our API (with capital letters), fixing the comment.